Cannot build latest fails with gulp sourcemap error

Make sure these boxes are checked before submitting your issue - thank you!

- [ * ] I have checked the superset logs for python stacktraces and included it here as text if any
- [ * ] I have reproduced the issue with at least the latest released version of superset
- [ * ] I have checked the issue tracker for the same issue and I haven't found one similar


### Superset version
Latest HEAD

### Expected results
npm install would install superset

### Actual results
npm ERR! fetch failed https://registry.npmjs.org/gulp-sourcemaps/-/gulp-sourcemaps-1.7.0.tgz

### Steps to reproduce
clone repository
`cd incubator-superset/superset/assets`
`npm install`

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,7 +10,7 @@ cache:
 env:
   global:
     - TRAVIS_CACHE=$HOME/.travis_cache/
-    - TRAVIS_NODE_VERSION="6.10.2"
+    - TRAVIS_NODE_VERSION="7.10.0"
   matrix:
     - TOX_ENV=javascript
     - TOX_ENV=pylint
@@ -19,7 +19,7 @@ env:
     - TOX_ENV=py27-mysql
     - TOX_ENV=py27-sqlite
 before_install:
-  - npm install -g npm@'>=4.5.0'
+  - npm install -g npm@'>=5.0.3'
 before_script:
   - mysql -e 'drop database if exists superset; create database superset DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci' -u root
   - mysql -u root -e "CREATE USER 'mysqluser'@'localhost' IDENTIFIED BY 'mysqluserpassword';"

--- a/superset/assets/js_build.sh
+++ b/superset/assets/js_build.sh
@@ -3,7 +3,8 @@ set -e
 cd "$(dirname "$0")"
 npm --version
 node --version
-npm install
+npm install -g yarn
+yarn
 npm run sync-backend
 npm run lint
 npm run test

